index.bs

```diff
@@ -2572,6 +2572,15 @@ Maturity Levels</h4>
 				An [=Obsolete Recommendation|Obsolete=] or Superseded specification
 				has the same status as a [=W3C Recommendation=]
 				with regards to W3C Royalty-Free IPR Licenses granted under the Patent Policy.
+
+				Note: When a Technical Report which had previously been published as a [=Recommendation=]
+				is again published as a [=Recommendation=]
+				(after the following the necessary steps to <a href="#revised-rec">revise</a> it),
+				it is implied that the previously published [=Recommendation=] has been superseded by the newer one:
```

Comment:
Hmm. That's not how I read the process. To me, it is indeed implied, because when you republish **the same** technical report, the latest publication replaces the previous ones (which are merely kept around for archival and documentation purposes). This is where the implication is: when you republish a report, this is always with the intent of replacing (i.e. superseding) previously published versions of the same report. And there isn't really an undo or a way to grant exceptions here; if you have published a document as CR, and decide that you liked the previous WD better, you cannot undo the publication of the CR (although you can publish a new WD that replaces the CR and has the same content as the old one). Similarly, If you have published a second REC of the same technical report, you cannot "un-supersede" the first REC without republishing it on top of the second one. In other words, a single technical report only has one latest version, and all the non-last versions are superseded by it.

Reply:
I think the potential confusion arises here because it is not clear what a new "version" is. Consider the following options:

1. GreatThing v1.0 REC -> GreatThing v1.0 REC with minor editorial errata fixed.
2. GreatThing v1.0 REC -> GreatThing v1.0 Second Edition REC with substantive errata fixed.
3. GreatThing v1.0 REC -> GreatThing v1.1 REC with new features

I think https://github.com/w3c/w3process/pull/213#discussion_r298822221 @frivoal unambiguously applies to scenario 1, and _probably_ applies to scenario 2, but it is unclear if it always applies to scenario 3. 

I would argue that the default for scenario 3 should be to supersede the previous version, but that the WG might reasonably request _not_ to supersede it, and that we need to say how they might make that request.